easybuild/easyconfigs/c/canu/canu-2.3-GCCcore-13.3.0-Java-11.eb

```diff
@@ -23,10 +23,16 @@ dependencies = [
     ('Java', '11', '', SYSTEM),
     ('Perl', '5.38.2'),
     ('gnuplot', '6.0.1'),
+    # mark next line if the environment has OpenSSL3
```

Comment:
this comment is unnecessary. if OpenSSL-3 is installed in the system the OpenSSL-3 installed `openssl` binary will be a symlink to the system installed one